.travis.yml

```diff
@@ -4,6 +4,7 @@ go:
 - 1.7.x
```

Comment:
I think we should drop 1.7.x right now and drop 1.8.x when 1.10 is released

Reply:
+1